Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/engineer.yml

```diff
@@ -246,7 +249,8 @@
       - id: RMCPouchFirstAidPills
         name: first-aid pouch (pills)
         recommended: true
-      #- id: CMElectronicsPouchFull
+      - id: RMCPouchElectronicsFill
```

Comment:
This one was commented out since there's no point to it yet, can't swap batteries for apcs and don't need to rebuild them